app/src/main/java/com/jc/android/plantdairy/notif/NotificationHelper.kt

```diff
@@ -0,0 +1,67 @@
+package com.jc.android.plantdairy.notif
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import com.jc.android.plantdairy.R
+import com.jc.android.plantdairy.ui.MainActivity
+
+object NotificationHelper {
+
+    fun createNotificationChannel(context: Context, importance: Int, showBadge: Boolean, name: String, description: String) {
+        // 1. Safety checked the OS version for API 26 and greater.
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            // 2. Created a unique name for the notification channel.
+            //    The name and description are displayed in the application’s Notification settings.
+            val channelId = "${context.packageName}-$name"
+            val channel = NotificationChannel(channelId, name, importance)
+            channel.description = description
+            channel.setShowBadge(showBadge)
+
+            // 3. Created the channel using the NotificationManager.
```

Comment:
When you use the code online, try to tweak it to make sure it doesn't like "other people" code. One thing you can do is delete this comment.